app/styles/ember-table.scss

```diff
@@ -43,10 +43,22 @@
     border-right: 1px solid var(--base03);
   }
 
+  .blue {
+    color: #174fff
```

Comment:
Minor: `;`

Reply:
Will fix in a follow up, and add stylelint to catch these things 👍